Test/git-issues/git-issue-3513/git-issue-3513.dfy

```diff
@@ -0,0 +1,8 @@
+// RUN: %exits-with 2 %baredafny resolve --use-basename-for-filename "%s" > "%t"
+// RUN: %exits-with 2 %baredafny resolve --use-basename-for-filename --library:"%S/src/A.dfy" "%s" >> "%t"
```

Comment:
The comma splitting behavior is not tested

Reply:
Added a test in a followup PR.